proto/money/rpc/reservation_folio_svc.proto

```diff
@@ -146,6 +146,21 @@ message ReservationFolioSvcVoidAuthorizationResponse {
 	ReservationFolioSvcVoidAuthorizationResult result = 1;
 }
 
+message ReservationFolioSvcPaymentCardRefundRequest {
+	holms.types.primitive.MonetaryAmount amount = 1;
+	holms.types.money.cards.PaymentCardIndicator card = 2;
+}
```

Comment:
You're going to want some handle on the folio or reservation here. The refund has to be accounted for on a booking at some point, and that booking's folio is also the source of our refund limit.